src/model/game/GameManager.ts

```diff
@@ -632,6 +632,12 @@ export default class GameManager {
             "https://thunderstore.io/c/castle-story/api/v1/package/", EXCLUSIONS,
             [new StorePlatformMetadata(StorePlatform.STEAM_DIRECT, "227860")], "CastleStory.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, ["cs"]),
+
+        new Game("Another Crab's Treasure", "AnotherCrabsTreasure", "AnotherCrabsTreasure",
+            "AnotherCrabsTreasure", ["AnotherCrabsTreasure.exe"], "AnotherCrabsTreasure_Data",
+            "https://thunderstore.io/c/another-crabs-treasure/api/v1/package/", EXCLUSIONS,
+            [new StorePlatformMetadata(StorePlatform.STEAM, "1887840")], "AnotherCrabsTreasure.png",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, []),
```

Comment:
Minor: the last empty array, used for sorthands for the text search in the game selection screen could contain "act". IDK if we have a convention for what shorthands to use, but initial letters of the game title would make sense to me.

Reply:
I remember Mythic saying "let's not use that"

Reply:
In this instance or as a general rule?